docker/archive/dest.go

```diff
@@ -19,17 +19,32 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
 		return nil, errors.Errorf("docker-archive: destination reference not supplied (must be of form <path>:<reference:tag>)")
 	}
-	fh, err := os.OpenFile(ref.path, os.O_WRONLY|os.O_EXCL|os.O_CREATE, 0644)
+	// tries to open a file first to accomodate for load when using stdout
+	// as the file is created first then written to
+	// if the file doesn't exist then it creates a new file for exclusive use
+	// if the file exists, it checks to ensure the size is 0
```

Comment:
It is curious that the documentation for docker load seems to indicate one image.
```
 docker load --help

Usage:	docker load [OPTIONS]

Load an image from a tar archive or STDIN

Options:
      --help           Print usage
  -i, --input string   Read from tar archive file, instead of STDIN
  -q, --quiet          Suppress the load output
```